tests/integration/abi_roundtrip_test.py

```diff
@@ -33,6 +34,15 @@
     "appl": GAI_ISSUE_2068,
 }
 
+
+class NamedTupleInherit(abi.NamedTuple):
+    a: abi.Bool
+    b: abi.Address
+    c: abi.Tuple2[abi.Uint64, abi.Bool]
+    d: abi.StaticArray[abi.Byte, Literal[10]]
+    e: abi.StaticArray[abi.Bool, Literal[4]]
```

Comment:
do 6?

Reply:
nah they won't let me, somewhere around it is restricted to generate `Tuple` type annotation, and we can go as far as 5.

I need to think a little more on how to get around.

Reply:
seems to break beyond the boundary of 5 right now, wanna see how it goes?